README.md

```diff
@@ -21,21 +21,21 @@ Below is a sample configuration file:
 
 ```toml
 client_mode = "MONITOR"
-#blacklist_regex = "^(?:/Users)/.*"
-#whitelist_regex = "^(?:/Users)/.*"
+#blocklist_regex = "^(?:/Users)/.*"
+#allowlist_regex = "^(?:/Users)/.*"
 batch_size = 100
 
 [[rules]]
 rule_type = "BINARY"
-policy = "BLACKLIST"
+policy = "blocklist"
```

Comment:
This (and the other changed entries in this file) should be all caps, [right](https://github.com/pmarkowsky/santa/blob/main/docs/development/sync-protocol.md#rules-objects)?

Reply:
Correct. Should be fixed now.